src/clean_split_data.py

```diff
@@ -0,0 +1,47 @@
+# author: MDS-2021-22 block3 group21
+# date: 2021-11-25
+"""
+Cleans NCDB 2017 data and creates training and test data set csv files.
+
+Usage: clean_split_data.py --input=<input filepath> --output=<output directory>
+
+Options:
+--input=<input filepath>    Filepath of data in csv format
+--output=<output directory> Directory specifying where to store training and test data sets
+"""  
+
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from docopt import docopt
+
+opt = docopt(__doc__)
+
+def main():    
+    # Importing NCDB 2017 dataset and dropping irrelevant or redundant columns
+    ncdb = (
+        pd.read_csv(opt["--input"], low_memory=False)
+        .sort_index()
+        .drop(columns=["C_YEAR", "C_CASE", "C_SEV"])
+    )
+
+    # Creating 'FATALITY' column to convert multi-class outcomes to binary-class
+    ncdb.loc[ncdb["P_ISEV"] == "3", "FATALITY"] = "True"
+    ncdb.loc[ncdb["P_ISEV"] != "3", "FATALITY"] = "False"
```

Comment:
if we are using True/False for the target variable, should we set its datatype boolean instead of string?

Reply:
That is a good point